src/main/java/DukeException.java

```diff
@@ -0,0 +1,12 @@
+/**
+ * Class representation of DukeException
+ */
+public class DukeException extends Exception{
+    /**
+     * DukeException constructor
+     * @param message
+     */
+    public DukeException(String message) {
```

Comment:
Perhaps you could add in more customisation here and add in more specific exceptions.